### PR TITLE
Add pendulum recipe

### PR DESCRIPTION
**Pull request by mcs07**
Depends on #3181

### CODE REVIEW

File: recipes/pendulum/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  features:
```
Comment (synapticarbors) [inline recipes/pendulum/meta.yaml]: Looking at the following wiki, it's not clear to me if you need the `features` section:

https://github.com/conda-forge/staged-recipes/wiki/VC-features

File: recipes/pendulum/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - vc 14  # [win and py>=35]
+  run:
+    - python
+    - python-dateutil
+    - pytzdata
+    - tzlocal
+    - vc 9  # [win and py27]
+    - vc 14  # [win and py>=35]
+
+test:
```
Comment (synapticarbors) [inline recipes/pendulum/meta.yaml]: With the compiled components, it might be good to run the test suite using pytest.

**Comment by ericmjl** [comment]
@synapticarbors surfacing up! Many apologies for leaving that PR hanging - a lot in life has changed since Nov 2016 (e.g. finally defending thesis!!!!). 

Is the previous PR preventing other builds from going forward? I'm happy for it to be killed - that's probably outdated by now.

**Comment by synapticarbors** [comment]
@ericmjl If you want to close the old PR that would be great. It's not causing any problems, but it is old and this PR will certainly supersede it. I was more curious if you'd like to be added as a co-maintainer on this recipe since you showed an interest in the package previously.

**Comment by ericmjl** [comment]
@synapticarbors thank you for considering me, though I think I'd like to pass (overcommitted on other conda-forge recipes, hope you can understand :smile:). I still think pendulum is an awesome package, though! Will definitely try to find more ways to use it, say, in personal projects or even other production stuff!

File: recipes/pendulum/meta.yaml (hunk 1 above)
Comment (isuruf) [inline recipes/pendulum/meta.yaml]: `vc` features and `vc` package are not needed. Relevant section is

> if it is a library that has compiled content, but is used only from Python, you need Python in both build and runtime requirements, but you do NOT need the features block.

**Comment by mcs07** [comment]
OK, vc features are removed

**Comment by mcs07** [comment]
I don't think the tests are included in the PyPI distribution, so it doesn't seem possible to run pytest?

**Pull request merged**